### DISCUSSION TITLE
kafka strimzi build own image

### DISCUSSION THREAD

**Discussion by chanvoyage** [strimzi/strimzi-kafka-operator#5321] [Q&A]
hello everyone

i want to build own images  and push to docker registry follow 
https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/DEV_GUIDE.md

I'm not sure what I miss something 

<img width="1440" alt="Screen Shot 2564-07-19 at 18 37 45" src="https://user-images.githubusercontent.com/39899497/126154424-218f0c85-4565-41b4-98c3-c5ba2508bd3e.png">

<img width="1164" alt="Screen Shot 2564-07-19 at 18 40 27" src="https://user-images.githubusercontent.com/39899497/126154759-94684b55-2d06-4f64-8aec-36886146223f.png">

**Comment by scholzj** [comment]
Did you build the whole project first? It looks like you are missing the Java builds.

**Reply by chanvoyage** [reply]
hello scholzj
Yes , first project  
i just know missing the java builds and i try to java builds found error during build 

<img width="1278" alt="Screen Shot 2564-07-20 at 13 25 47" src="https://user-images.githubusercontent.com/39899497/126277282-b4d56d4a-e767-4fd4-9813-8af4a9465eea.png">

<img width="1440" alt="Screen Shot 2564-07-20 at 14 23 03" src="https://user-images.githubusercontent.com/39899497/126279066-6d4ec984-cd50-47f6-b0d3-3d1c6a5cfae9.png">



[error.log](https://github.com/strimzi/strimzi-kafka-operator/files/6846198/error.log)

**Reply by scholzj** [reply]
I guess you might be missing some dependency or might have wrong version of Java? Do you use OpenJDK 11? Do you have all the dependencies such as `yq`, etc.?

**Reply by chanvoyage** [reply]
hi scholzj

i fix problem  change java from java 8 to java 11 , i think  class file not support java 8 

thank you very much

i have some question if i want to build image with jenkins pipline Do you have any suggestion  or  document guideline for me ?

**Reply by scholzj** [reply]
Yeah, we don't support Java 8 anymore.
